integration/spark/src/main/scala/org/apache/spark/sql/events/MergeIndexEventListener.scala

```diff
@@ -43,11 +43,6 @@ class MergeIndexEventListener extends OperationEventListener with Logging {
   override def onEvent(event: Event, operationContext: OperationContext): Unit = {
     event match {
       case preStatusUpdateEvent: LoadTablePreStatusUpdateEvent =>
-        // skip merge index in case of insert stage flow
-        if (null != operationContext.getProperty(CarbonCommonConstants.IS_INSERT_STAGE) &&
```

Comment:
This property also has to be removed from CarbonCommonConstants, as no more usage will be found

Reply:
I have modified code according to your suggestion